checkstyle.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE module PUBLIC
+				"-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+				"http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<module name="Checker">
+	<property name="charset" value="UTF-8"/>
+
+	<module name="TreeWalker">
+		<property name="fileExtensions" value="java"/>
+		<property name="tabWidth" value="1"/>
+
+		<!-- Force clean imports (no redundant and unused imports) -->
+		<module name="RedundantImport"/>
+		<module name="UnusedImports"/>
+
+		<!-- Force tab indentation -->
+		<module name="RegexpSinglelineJava">
+			<property name="format" value="^\t* "/>
+			<property name="message" value="Indent must use tab characters"/>
+			<property name="ignoreComments" value="true"/>
+		</module>
+
+		<module name="OverloadMethodsDeclarationOrder"/>
+
+		<!--<module name="AvoidStarImport"/>-->
```

Comment:
I would remove this line to keep it as clear as we can.

Reply:
Yep, I wanted to discuss the rule actually :) I'd prefer no star imports personally.

Reply:
Fine by me... Even might be a little bit annoying with Idea that automatically uses star imports. But that's configurable I believe.